packages/web3-eth-contract/test/unit/contract.test.ts

```diff
@@ -755,7 +756,7 @@ describe('Contract', () => {
 
 			const clonnedContract = contract.clone();
 
-			expect(JSON.stringify(contract)).toStrictEqual(JSON.stringify(clonnedContract));
+			expect(stringify(contract)).toStrictEqual(stringify(clonnedContract));
```

Comment:
eventemitter3 has a circular structure so we use stringify library to parse properly

Reply:
whats error for that?

Reply:
```
    TypeError: Converting circular structure to JSON
        --> starting at object with constructor 'EventEmitter'
```